### PR TITLE
fix yarn pnpn

### PR DESCRIPTION
**Pull request by Eomm**
https://github.com/fastify/fastify/runs/2638624327
our types use the ajv types so we must ship it as dependency

Customizing the ajv version to v8, seems impossible for typescript's users so

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -178,6 +178,7 @@
     "@fastify/ajv-compiler": "^1.0.0",
     "@fastify/proxy-addr": "^3.0.0",
     "abstract-logging": "^2.0.0",
+    "ajv": "^6.12.2",
```
Comment (jsumners) [inline package.json:181]: Can we fix the issues without re-adding this dependency? If not for v3 then for v4?

Comment (mcollina) [reply package.json:181]: I think so but it will require some reorg of our types.

Comment (climba03003) [reply package.json:181]: We should export `ajv` types from `@fastify/ajv-compiler`, and we can have a the appropriate types for `ajv`.
Let say `@fastify/ajv-compiler@1` will export `ajv@6`, `@fastify/ajv-compiler@1` will export `ajv@8`.

Comment (Eomm) [reply package.json:181]: Could you send a PR there or guide me on how to do that?

Comment (climba03003) [reply package.json:181]: @Eomm https://github.com/fastify/ajv-compiler/pull/12

**Comment by mcollina** [comment]
ajv-compiler@1.1.0 with the types has been released.

**Comment by mcollina** [comment]
here is the PR to fix it https://github.com/fastify/fastify/pull/3088

**Pull request closed without merging**